raco/backends/sql/catalog.py

```diff
@@ -169,6 +172,9 @@ def _get_unary_sql(self, plan):
                 return sel
             return sel.group_by(*g)
 
+        elif isinstance(plan, algebra.Distinct):
```

Comment:
I want to be sure I understand how this translates to SQL. So something like
`[from R emit distinct foo, bar]`
translates to
```
SELECT DISTINCT S.* FROM (
  SELECT foo, bar FROM R
) AS S;
```
?